### DISCUSSION TITLE
Proposal for regular agenda change to Community Working Group

### DISCUSSION THREAD

**Discussion by adamj89** [Green-Software-Foundation/discussions#39] [Community Working Group]
Proposal from @jawache to dedicate half (30 Mins) the regular CWG meeting to internal advocacy. Each fortnight having a different member company take a 30 min slot with the PPP (Progress, Plans & Problems) in building their internal green software community. A mixture of experienced and new companies. 

The back half-hour of each meeting would cover the other CWG projects and discussions.

**Comment by tmcclell** [comment]
Discussions around original charter, discussion around grouping companies by industry. Create a roadmap of presenting companies, once monthly choose topics for more details (i.e., training) lightening talk and discussion. Proposal to trial new format, agenda - create roadmap and topics for alternative meetings.  Communicate new format to members expressing interest. Proposal was approved.
